plugins/package-managers/pub/src/main/kotlin/utils/PubCacheReader.kt

```diff
@@ -36,7 +35,7 @@ import org.ossreviewtoolkit.utils.common.textValueOrEmpty
  * directory. If Flutter is installed it additionally looks for files in the ".pub-cache" directory of Flutter's
  * installation directory.
  */
-internal class PubCacheReader {
+internal class PubCacheReader(flutterHome: File? = null) {
```

Comment:
I have added an explanation to the commit message:

    This change requires to pass the `flutterHome` variable as a constructor
    argument to the `PubCacheReader`. Make this argument nullable, because
    it was ignored anyway if it did not contain a `.pub-cache` subdirectory,
    and to not have to provide it when testing the `PubCacheReader`.

I have also dropped the last commit with the version update and will make a separate PR for that because I won't be able to update the expected results today.